spec/lib/diaspora_federation/receiver/private_spec.rb

```diff
@@ -0,0 +1,64 @@
+module DiasporaFederation
+  describe Receiver::Private do
+    let(:sender_id) { FactoryGirl.generate(:diaspora_id) }
+    let(:sender_key) { OpenSSL::PKey::RSA.generate(1024) }
+    let(:recipient_key) { OpenSSL::PKey::RSA.generate(1024) }
+    let(:recipient_guid) { FactoryGirl.generate(:guid) }
+    let(:xml) {
+      DiasporaFederation::Salmon::EncryptedSlap.generate_xml(
+        sender_id,
+        sender_key,
+        FactoryGirl.build(:request_entity),
+        recipient_key
+      )
+    }
+
+    it "calls entity_persist if everyting is fine" do
+      expect(DiasporaFederation.callbacks).to receive(:trigger)
+                                                .with(:fetch_public_key_by_diaspora_id, sender_id)
+                                                .once
```

Comment:
Then why do they need the specifier? I though default is any times.
There is no specifier for that.